runtime/parachains/src/scheduler_polkadot/mod.rs

```diff
@@ -74,17 +72,19 @@ impl<T: Config> AssignmentProvider<T> for Pallet<T> {
 		}
 	}
 
-	fn push_assignment_for_core(core_idx: CoreIndex, assignment: Assignment) {
-		let parachain_cores = NumParachains::<T>::get()
-			.unwrap_or_else(|| <crate::scheduler_parachains::Pallet<T>>::session_core_count());
-		if (0..parachain_cores).contains(&core_idx.0) {
-			<crate::scheduler_parachains::Pallet<T>>::push_assignment_for_core(core_idx, assignment)
-		} else {
-			let _core_idx = CoreIndex(core_idx.0 - parachain_cores);
-			todo!()
-			//<crate::scheduler_parathreads::Pallet<T>>::push_assignment_for_core(
-			//	core_idx, assignment,
-			//)
+	fn push_parasentry_for_core(core_idx: CoreIndex, entry: ParasEntry) {
+		if entry.retries == 0 {
```

Comment:
As mentioned earlier, I don't think the assignment provider should be concerned with retries.